sources/packages/backend/apps/workers/src/controllers/supporting-user/supporting-user.controller.ts

```diff
@@ -61,7 +69,33 @@ export class SupportingUserController {
         (supportingUser) => supportingUser.id,
       );
       jobLogger.log("Created supporting users.");
-      return job.complete({ createdSupportingUsersIds });
+      return this.dataSource.transaction(
+        async (transactionalEntityManager: EntityManager) => {
```

Comment:
I see that createSupporting users is done outside the transaction. The intention of using the transaction is createSupportingUsers and saveNotification must be executed in a single commit.